src/main.rs

```diff
@@ -53,6 +53,9 @@ struct FdOptions {
     /// Whether to follow symlinks or not.
     follow_links: bool,
 
+    /// Whether elements of output should be separated by a null character
+    zero: bool,
```

Comment:
I find the name `zero` a little bit too generic here. What do you think about just naming this `null_separator` or `print0`? Another option would be to call this `output_separator` and use an enum `OutputSeparator { Newline, NullCharacter }`